lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -293,6 +293,9 @@ class SwiftASTContext : public TypeSystemSwift {
   swift::ModuleDecl *CreateModule(const SourceModule &module, Status &error,
                                   swift::ImplicitImportInfo importInfo);
 
+  /// Generic callback function used for progress reporting that gets
+  /// invoked by the Swift compiler and gets set upon scope exit when
+  /// a more detailed progress report was used as the callback
   static bool ReportModuleLoadingProgress(llvm::StringRef module_name,
                                           bool is_overlay);
```

Comment:
This is still here ...

Reply:
Removed, I thought I took this out of the header 😓